sa/db-next/boulder_sa/20250417000000_RateLimitOverrides.sql

```diff
@@ -0,0 +1,20 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE overrides (
+   `limitEnum` tinyint(4) UNSIGNED NOT NULL,
+   `bucketKey` varchar(255) NOT NULL,
+   `comment`   varchar(255) DEFAULT NULL,
+   `periodNS`  bigint(20) UNSIGNED NOT NULL,
+   `count`     int UNSIGNED NOT NULL,
+   `burst`     int UNSIGNED DEFAULT NULL,
```

Comment:
why is this one DEFAULT NULL instead of NOT NULL?

Reply:
Good catch, I was originally considering an implementation that defaulted burst to count but I didn't end up going that way.